example/use_cases/employer_university_flow/employer_university_flow/issuer.go

```diff
@@ -0,0 +1,83 @@
+package employer_university_flow
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/sirupsen/logrus"
+)
+
+func BuildExampleUniversityVC(universityID string, recipient string) (*credential.VerifiableCredential, error) {
```

Comment:
this doesn't have to be changed, but in case you didn't know if multiple parameters have the same type you can do this

`BuildExampleUniversityVC(universityID, recipient string)`

Reply:
yep. I'm aware. Good catch yough.